.github/workflows/ci.yml

```diff
@@ -12,39 +12,35 @@ jobs:
     strategy:
```

Comment:
Updating and simplifying this. Dropping 2.3 as it's EOL and not available in the CI images.